docs/get-started/Dockerfile.cuda

```diff
@@ -0,0 +1,86 @@
+#
+# Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+FROM nvidia/cuda:10.1-devel-ubuntu18.04 
+ARG spark_uid=185
+
+# Install java dependencies 
+RUN apt-get update && apt-get install -y --no-install-recommends openjdk-8-jdk openjdk-8-jre
+ENV JAVA_HOME /usr/lib/jvm/java-1.8.0-openjdk-amd64
+ENV PATH $PATH:/usr/lib/jvm/java-1.8.0-openjdk-amd64/jre/bin:/usr/lib/jvm/java-1.8.0-openjdk-amd64/bin
+
+# Before building the docker image, first build and make a Spark distribution following
+# the instructions in http://spark.apache.org/docs/3.0.1/building-spark.html
```

Comment:
I was thinking instead of telling them to build we should just tell them to download the release.   http://spark.apache.org/downloads.html. We could say something about - go download or build spark 3.0.1

Reply:
Honestly if we wanted to make this really easy we could change the docker file to just fetch it. let me know if you want me to try that out.

Reply:
I updated the instructions to point to a download location.